### PR TITLE
ref(integrations): Better default issue description

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

**Review by billyvg — APPROVED** [review]
`Associated Sentry Issue` seems a bit wordy

File: src/sentry/integrations/issues.py [hunk 1]
```diff
@@ -31,7 +31,10 @@ def get_group_body(self, group, event, **kwargs):
 
     def get_group_description(self, group, event, **kwargs):
         output = [
-            absolute_uri(group.get_absolute_url()),
+            u'Associated Sentry Issue: [{}]({})'.format(
```
Comment (macqueen) [inline src/sentry/integrations/issues.py]: is this formatting going to work for jira too?

Comment (EvanPurkhiser) [reply src/sentry/integrations/issues.py]: See https://github.com/getsentry/sentry/pull/9295

**Comment by EvanPurkhiser** [comment]
I wouldn't mind better wording. /cc @saragilford

**Review by macqueen — APPROVED** [review]
